CSharpSourceCode/Battle/Dismemberment/Dismemberment.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using TaleWorlds.Core;
+using TaleWorlds.Engine;
+using TaleWorlds.InputSystem;
+using TaleWorlds.Library;
+using TaleWorlds.MountAndBlade;
+
+namespace TOW_Core.Battle.Dismemberment
+{
+    public class Dismemberment
+    {
+        public static void DismemberHead(Agent victim, AttackCollisionData attackCollision)
+        {
+            //if (victim.IsActive())
+            //    KillAgent(victim);
+            victim.AgentVisuals.SetVoiceDefinitionIndex(-1, 0f);
+            MakeHeadInvisible(victim);
+            GameEntity head = SpawnHead(victim);
+            AddPhysicsFromCollision(head, attackCollision);
+            CreateBloodBurst(victim);
```

Comment:
agree that could be all inside the same mission logic. I dont understand the difference here. I would move the class if the Inheritage of another tale worlds class , is needed otherwise , if the function are that related, keep them together